README.md

```diff
@@ -159,24 +159,15 @@ Assume the current `starr` version is `2.12.0` (defined in
     generating API docs, see above).
   - Quit sbt and start a new sbt instance using `sbt -Dstarr.version=<version>` where
     `<version>` is the version number you published locally.
-  - If the version number you published is not binary compatible with the current
-    `starr`, `set every scalaBinaryVersion := "2.12.0-M4"`. This is not required if
-    the version you published locally is binary compatible, i.e., if the current
-    `starr` is a 2.12.x release and not a milestone / RC.
-
-The last step is required to resolve modules (scala-partest, etc). It
-assumes that the module releases for the current `starr` work (in terms of binary
-compatibility) with the local starr that you published locally. A full bootstrap
-requires re-building the all the modules. On our CI this is handled by the
-[bootstrap](scripts/jobs/integrate/bootstrap) script, but it (currently) cannot
-be easily executed locally.
 
 ### IDE setup
 
-You may use IntelliJ IDEA (see [src/intellij/README.md](src/intellij/README.md)),
-or ENSIME (see [this page on the ENSIME site](https://ensime.github.io/editors/)).
-(Support for Eclipse has decayed and been removed, but could be resurrected by
-a volunteer.)
+We suggest using IntelliJ IDEA (see
+[src/intellij/README.md](src/intellij/README.md).
+
+(Metals should also work, but we don't yet have instructions or sample
```

Comment:
I'm sure most people who get this far have heard of it, but a metals link wouldn't hurt (and ENSIME got one!)